src/components/Notifications.js

```diff
@@ -39,32 +38,7 @@ const Notifications = () => {
 
   useEffect(() => {
     const fetchNotifications = async () => {
-      const response = await axios.post(
-        `${apiURL}/graphql`,
-        {
-          query: `query {
-              userStoryNotifications (where: {
-                users: {
-                  id: "${userId}"
-                }
-              }){
-                message
-                id
-                users {
-                  id
-                }
-                seenBy {
-                  id
-                }
-                date
-                link
-              }
-            }`
-        },
-        {
-          withCredentials: true
-        }
-      )
+      const response = await userStory.getNotificationsComponent(userId)
```

Comment:
Can we rename this to `getNotificationsByUserId`?

Reply:
Done!